src/components/Patient/MedicationStatementList.tsx

```diff
@@ -45,9 +44,9 @@ export function MedicationStatementList({
   status = MEDICATION_STATEMENT_STATUS.filter(
     (status) => status !== "entered_in_error",
   ),
+  readOnly = false,
```

Comment:
pass in canAccess into EncounterAccordianLayout; remove readOnly prop for MedicationStatementList.